gr-qtgui/lib/eye_sink_c_impl.cc

```diff
@@ -188,7 +188,7 @@ void eye_sink_c_impl::set_samp_per_symbol(unsigned int sps)
 
 void eye_sink_c_impl::set_title(const std::string& title)
 {
-    // set_title no longer used but called by swig
```

Comment:
Can set_title be removed altogether?

Reply:
Yes, but it touches quite a few files. I'm thinking it should be a separate pull request.